generated/nifake/nifake/session.py

```diff
@@ -697,8 +697,8 @@ def __init__(self, resource_name, options={}, id_query=False, reset_device=False
 
         '''
         if _grpc_channel:
-            import nifake._grpc as _grpc
-            library_interpreter = _grpc.GrpcStubInterpreter(_grpc_channel)
+            import nifake._grpc_stub_interpreter as _grpc_stub_interpreter
+            library_interpreter = _grpc_stub_interpreter.GrpcStubInterpreter(_grpc_channel)
```

Comment:
Should the session's member now just be called `interpreter`? Note the current name is now incorrect for gRPC.

Reply:
Consider: naming the member `implementation` might be more human readable.

Reply:
FYI @ni-jfitzger

Reply:
I agree that the name `library_interpreter` doesn't fit the gRPC case. I don't feel great about the name `implementation`. It's not an object of some child class of an interface class and `implementation` also feels pretty vague.

`interpreter` feels like a good compromise.

Do we need an ABC Interpreter class for each of the interpreters to inherit from?

I'm interested in hearing @marcoskirsch's thoughts.

Reply:
I definitely think the varaible should be called `interpreter` and the member `_interpreter`.

Reply:
I'm ok if for noisiness @DavidCurtiss prefers to do it in a separate PR.

Reply:
It _is_ a bit noisy, but it's straightforward, and won't clash with the file rename, so I'll go ahead and do it here, and reset Jay's vote.

Reply:
Done